### PR TITLE
fix the call to loadNuxtConfig by passing empty object

### LINKED ISSUE #7375: Nuxt3 with Web Frameworks fails to init or deploy - Cannot create property 'overrides' on string
### [REQUIRED] Environment info

**firebase-tools:** 13.11.4

**Platform:** macOS

### [REQUIRED] Test case

https://github.com/JamieCurnow/firebase-web-framework-nuxt-loadNuxtConfig

### [REQUIRED] Steps to reproduce

This is a fresh nuxt install that was created using `npx nuxi@latest init`

Open the repo.
Run `npm i` to install deps.
Run `firebase init --debug`
Choose only `Hosting: Configure files for Firebase Hosting and (optionally) set up GitHub Action deploys`.
Use and existing project or create a new one.

### [REQUIRED] Expected behavior

Firebase init sets up the firebase project for the repo.

### [REQUIRED] Actual behavior

The command fails due to a recent(ish) update in nuxt/kit.

```
TypeError: Cannot create property 'overrides' on string '/Users/JCGeek/Documents/Projects/firebase-tools-issue/nuxt-app'
    at loadNuxtConfig (file:///Users/JCGeek/Documents/Projects/firebase-tools-issue/nuxt-app/node_modules/@nuxt/kit/dist/index.mjs:2581:18)
    at async getConfig (/Users/JCGeek/.nvm/versions/node/v20.14.0/lib/node_modules/firebase-tools/lib/frameworks/nuxt/index.js:92:12)
    at async discover (/Users/JCGeek/.nvm/versions/node/v20.14.0/lib/node_modules/firebase-tools/lib/frameworks/nuxt/index.js:28:37)
    at async discover (/Users/JCGeek/.nvm/versions/node/v20.14.0/lib/node_modules/firebase-tools/lib/frameworks/index.js:47:32)
    at async doSetup (/Users/JCGeek/.nvm/versions/node/v20.14.0/lib/node_modules/firebase-tools/lib/init/features/hosting/index.js:54:11)
    at async init (/Users/JCGeek/.nvm/versions/node/v20.14.0/lib/node_modules/firebase-tools/lib/init/index.js:39:9)
```

### Research

It looks like `firebase-tools` is calling `loadNuxtConfig(dir)` [here](https://github.com/firebase/firebase-tools/blob/a102820460e25a084bef69e92bce31980577ad02/src/frameworks/nuxt/index.ts#L111) and passing in `dir` as a string.
In a recent Nuxt update that was made in PR [here](https://github.com/nuxt/nuxt/pull/27221/files#diff-7ad8f630a47db89cb01864c3fda27523d4b7cebad9c1129c0f78499183b5a7adR17) loadNuxtConfig now requires an object to be passed in.

### Quick Fix

Downgrade nuxt to v3.11.2 - `npm i nuxt@3.11.2`, and explicitly install @nuxt/kit at 3.11.2 - `npm i -D @nuxt/kit@3.11.2`.
Run `firebase init` and it should work.

### Actual fix

Pass an object through to loadNuxtConfig() - it seems as though passing the dir through has no affect at least in firebase init.

```ts
async function getConfig(dir) {
  const { loadNuxtConfig } = await (0, utils_1.relativeRequire)(dir, "@nuxt/kit")
  return await loadNuxtConfig({})
}
```

I'll test a deploy and put up a PR.

### PR DESCRIPTION
**Pull request by JamieCurnow**
### Description

Fixes #7375 - a recent update of nuxt has prevented deployments as described in the issue. @nuxt/kit used to just spread the single arg passed into loadNuxtConfig() which would have been a string `dir`, so this issue hasn't appeared as you can spread a string, even though it would have had no affect on functionality. A recent update actually uses the first arg to this function and expects it to be an object. This PR fixes the type error that was caused by passing a string when an object was expected.

### Scenarios Tested

I've done a fresh install of nuxt, a `firebase init` and a `firebase deploy` all of which now work with this change.

### CODE REVIEW

File: src/frameworks/nuxt/index.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ export async function getDevModeHandle(cwd: string) {
 
 export async function getConfig(dir: string): Promise<NuxtOptions> {
   const { loadNuxtConfig } = await relativeRequire(dir, "@nuxt/kit");
-  return await loadNuxtConfig(dir);
+  return await loadNuxtConfig({});
```
Comment (leoortizz) [inline src/frameworks/nuxt/index.ts:111]: Can you please apply this change here so we still pass the project directory to `loadNuxtConfig`? 
https://github.com/firebase/firebase-tools/blob/c496f61cb683ce2cbb9f8df4196de79efc690cb6/src/frameworks/nuxt/index.ts#L129-L133

I also have this fix in #7297 but that's not merged yet for other reasons.

**Review by leoortizz — CHANGES_REQUESTED** [review]
thanks @JamieCurnow!

File: src/frameworks/nuxt/index.ts (hunk 1 above)
Comment (JamieCurnow) [reply src/frameworks/nuxt/index.ts]: Great idea @leoortizz 👍 I've just pushed that up

**Review by leoortizz — COMMENTED** [review]
LGTM! Please add changelog then we can merge

**Comment by JamieCurnow** [comment]
Changelog added @leoortizz 🙌

**Review by leoortizz — APPROVED** [review]

**Comment by leoortizz** [comment]
@JamieCurnow looks like prettier is failing for the changelog file, can you make sure to run prettier on it?

**Comment by JamieCurnow** [comment]
Sorry about that @leoortizz - all fixed now and master merged back in 👌

**Comment by leoortizz** [comment]
> Sorry about that @leoortizz - all fixed now and master merged back in 👌

 Thank you 🙏 . We couldn't merge yet and we have a merge conflict now, can you please fix?

**Review by leoortizz — APPROVED** [review]

**Comment by JamieCurnow** [comment]
@leoortizz I've just fixed the merge conflict with the changelog again 👌

**Comment by leoortizz** [comment]
cc @jamesdaniels, this PR isn't being merged because the integration tests are being skipped for some reason, even if I  manually run them, any clue of what's missing?

**Pull request merged**